server/config/validation/add-smartcard-spend.js

```diff
@@ -18,6 +18,11 @@
             summary: 'Smartcard spend can only be 0 or more',
             details: 'Smartcard spend can only be 0 or more',
           },
+          lessThanOrEqualTo: 1000000,
+          notLessThanOrEqualTo: {
+            summary: 'Smartcard spend can not exceed 1,000,000',
```

Comment:
The wording we're using elsewhere is `must be less than £1,000,000`, I don't remember if that was a carefully worded string from design, or if we just made it up though :)